modules/claims_api/lib/bgs_service/manage_representative_service.rb

```diff
@@ -1,18 +1,53 @@
 # frozen_string_literal: true
 
-require_relative 'manage_representative_service/update_poa_request'
-
 module ClaimsApi
   class ManageRepresentativeService < ClaimsApi::LocalBGS
-    private
-
-    def make_request(**args)
-      super(
-        endpoint: 'VDC/ManageRepresentativeService',
-        namespaces: { 'data' => '/data' },
-        transform_response: false,
-        **args
-      )
+    def bean_name
+      'VDC/ManageRepresentativeService'
+    end
+
+    def read_poa_request(poa_codes: [])
+      # Workaround to allow multiple roots in the Nokogiri XML builder
+      # https://stackoverflow.com/a/4907450
+      doc = Nokogiri::XML::DocumentFragment.parse ''
+
+      Nokogiri::XML::Builder.with(doc) do |xml|
+        xml.send('data:POACodeList') do
+          poa_codes.each do |poa_code|
+            xml.POACode poa_code
+          end
+        end
+        xml.send('data:SecondaryStatusList') do
+          %w[New Pending Accepted Declined].each do |status|
+            xml.SecondaryStatus status
+          end
+        end
```

Comment:
Including all the statuses for the MVP, but we can add filters to the request as needed.